packages/idempotency/README.md

```diff
@@ -163,6 +164,69 @@ export const handler = makeIdempotent(myHandler, {
 
 Check the [docs](https://docs.powertools.aws.dev/lambda/typescript/latest/utilities/idempotency/) for more examples.
 
+### Decorator
+
+You can make any function idempotent, and safe to retry, by decorating it using the `@idempotent` decorator.
+
+```ts
+import { idempotent } from '@aws-lambda-powertools/idempotency';
+import { LambdaInterface } from '@aws-lambda-powertools/commons';
+import { DynamoDBPersistenceLayer } from '@aws-lambda-powertools/idempotency/dynamodb';
+import type { Context, APIGatewayProxyEvent } from 'aws-lambda';
+
+const persistenceStore = new DynamoDBPersistenceLayer({
+  tableName: 'idempotencyTableName',
+});
+
+class MyHandler extends LambdaInterface {
+  @idempotent({ persistenceStore: dynamoDBPersistenceLayer })
+  public async handler(
+          event: APIGatewayProxyEvent,
+          context: Context
+  ): Promise<void> {
+    // your code goes here here

```

Comment:
I'd suggest to return something here.

Reply:
Nah, let's keep it consistent with `makeIdempotent`